src/samples/EducationStart/LandingPage.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useContext, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { useTranslation } from 'react-i18next';
+import MainWrapper from '../../components/BaseComponents/MainWrapper';
+import Button from '../../components/BaseComponents/Button/Button';
+import AppHeader from './reuseables/AppHeader';
+import AppFooter from '../../components/AppComponents/AppFooter';
+import useHMRCExternalLinks from '../../components/helpers/hooks/HMRCExternalLinks';
+// import setPageTitle from '../../components/helpers/setPageTitleHelpers';
+import AppContext from './reuseables/AppContext';
+import { loginIfNecessary } from '@pega/auth/lib/sdk-auth-manager';
+import { useStartMashup } from './reuseables/PegaSetup';
+import { triggerLogout } from '../../components/helpers/utils';
+
+export default function LandingPage(props) {
+  const { onProceedHandler } = props;
+  const { showLanguageToggle } = useContext(AppContext);
+
+  const { hmrcURL } = useHMRCExternalLinks();
+  const history = useHistory();
+  const setAuthType = useState('gg')[1];
+
+  const { operatorName } = useStartMashup(
+    setAuthType,
+    // eslint-disable-next-line @typescript-eslint/no-use-before-define
+    doRedirectDone,
+    { appBacklinkProps: {} }
+  );
+
+  const { t } = useTranslation();
+
+  const onContinue = () => {
+    onProceedHandler();
+  };
+
+  function doRedirectDone() {
+    history.push('/education/start');
+    // appName and mainRedirect params have to be same as earlier invocation
+    loginIfNecessary({ appName: 'embedded', mainRedirect: true });
+  }
+
+  function handleSignout() {
+    triggerLogout();
+  }
+
+  return (
+    <>
+      <AppHeader
+        handleSignout={handleSignout}
+        appname={t('EDUCATION_START')}
+        hasLanguageToggle={showLanguageToggle}
+        betafeedbackurl={`${hmrcURL}contact/beta-feedback?service=463&referrerUrl=${window.location}`}
+      />
+      <div className='govuk-width-container'>
+        <MainWrapper>
+          <span className='govuk-caption-xl'>{operatorName}</span>
+          <h1 className='govuk-heading-xl'>
+            Extend your Child Benefit payments while a young person stays in education
```

Comment:
add the text content to language json. 
Create namespace for education start if required https://www.i18next.com/principles/namespaces